charts/keycloak/templates/keycloak_db.yaml

```
@@ -1,5 +1,18 @@
 {{- if .Values.keycloak.database.create }}
 ---
+{{- if .Values.keycloak.pvc.enabled }}
```

```suggestion
{{- if .Values.keycloak.database.pvc.enabled }}
```

Since this PVC is used only by the database, the values should be inside the key database for clarity.